atef/ui/run_check.ui

```diff
@@ -27,14 +27,21 @@
   </property>
```

Comment:
Agreed on setting up a spacer between the last widget and the RunCheck widget, I can do that here easily enough.  

I think I originally placed it in the bottom as to avoid conflicting with the name-description-tags widget when autogenerating a run-page.  In that context I like it there.  

Do you mean somewhat outside the page (outside the tab widget container)?  more in the window than the page?

Reply:
I guess top or bottom, either would be fine as far as I'm concerned. 

A consistent spot where I can always expect to click 'next' without moving my mouse around is what I'd hope for.

Reply:
In my setup the next button is in a consistent place (once the RunCheck widget is spaced to the bottom), is it not for you?  Does it shift left/right?

Reply:
Is this pulling us slowly in the direction of _always_ having some sort of prev/next navigation bar at the top or bottom rather than adding it to the page widgets as needed?

Reply:
Not horizontally, just vertically in the spacer-less case right now. I think what I was after is that the "Page" can be pretty large and we may end up putting a scroll area there. In which case, I would not want to have to scroll to see the 'prev/next' buttons, but rather have it part of the parent layout. (This is all a little difficult to explain in words - and a bit forward-looking. I think it's perfectly fine to wait to address this and see how you think it works best with the remainder of the config widgets and all)

> Is this pulling us slowly in the direction of _always_ having some sort of prev/next navigation bar at the top or bottom rather than adding it to the page widgets as needed?

I like the idea of always having prev/next, verify/run/status/etc can be selectively shown depending on the mode